docs/_data/atlantic-2023-schedule.yaml

```diff
@@ -0,0 +1,112 @@
+writing_day:
+  - time: '13:00'
+    title: 'Platform opens'
+    duration: '01:00'
+  - duration: '01:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - duration: '00:30'
+    title: Team up and start working
+  - duration: '01:30'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '01:30'
+    title: '🍲 Lunch Break - Determine length with table'
+  - duration: '03:00'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '00:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '13:00'
+    title: Platform opens
+    duration: '00:15'
+  - duration: '00:45'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '00:45'
+    title: Write the Docs Team - Community update
+  - duration: '0:05'
+    title: '⏲ 5 minute break'
+  - duration: '0:00'
+    title: "<b>Unconference Starts</b>"
+  - duration: '0:30'
+    slug: working-with-user-experience-to-maximize-user-success-katie-riker
+  - duration: '0:05'
+    title: '⏲ 5 minute break'
+  - duration: '0:30'
+    slug: one-glossary-to-rule-them-all-terminology-management-at-scale-ian-cowley
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '0:15'
+    title: '☕️ Coffee break'
+  - duration: '0:30'
+    slug: papercuts-stop-the-bleed-reducing-information-leakage-from-client-bound-docume-kristine-sihto
+  - duration: '0:05'
+    title: '⏲ 5 minute break'
+  - duration: '0:30'
+    slug: graphs-not-trees-a-ground-up-approach-to-fixing-a-docs-site-matt-dodson
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '1:00'
+    title: '🍲 Lunch Break'
+  - duration: '0:30'
+    title: ⚡️ Lightning Talks (30 mins)
```

Comment:
I might either add a break after lightning talks, or make them 35-40 mins in duration to handle swapping time.

Reply:
Done.